ion/tokenizer.go

```diff
@@ -1263,3 +1273,24 @@ func (t *tokenizer) unread(c int) {
 	t.pos--
 	t.buffer = append(t.buffer, c)
 }
+
+func isInvalidChar(c int) bool {
+	if c < 0x00 || c > 0x1F {
+		return false
+	}
+	if isWhiteSpaceChar(c) || isNewLineChar(c) {
+		return false
+	}
+	return true
+}
+
+func isWhiteSpaceChar(c int) bool {
```

Comment:
`isWhitespace` is similarly-named but works a bit differently. Maybe worth combining them? Otherwise maybe rename this one to `isStringWhitespace` or something like that.